### DISCUSSION TITLE
How to insert data from GeoJson?

### DISCUSSION THREAD

**Discussion by emd-che** [clickbar/laravel-magellan#60] [Q&A] [answered]
Hey everyone! I'm  really grateful for this awesome library, and I would really appreciate the help.

My question is how to insert data from GeoJson? can you help with an example?

Thanks,

**Comment by studnitz** [comment, marked as answer]
If you are handling `GeoJSON` in your `FormRequest`s you can use the [`TransformsGeojsonGeometry`](https://github.com/clickbar/laravel-magellan#request-validation-and-transformation) Trait or you can use the [`GeojsonParser`](https://github.com/clickbar/laravel-magellan/blob/main/src/IO/Parser/Geojson/GeojsonParser.php) class directly.

**Comment by ahawlitschek** [comment]
I guess you are receiving the GeoJson from a client via request. 
What does the actual GeoJson your sending look like?

Are you sending Geometries in GeoJson Format or a whole FeatureCollection?

**Reply by emd-che** [reply]
I would like to receive the GeoJSON data from a client through an uploaded file in GeoJSON format, and then use this data to update the existing table in the database with the corresponding geometries.
